{{ cookiecutter.package_name }}/{{ cookiecutter.module_name }}/_{{ cookiecutter._parent_project }}_init.py

```diff
@@ -17,36 +17,9 @@
 
 if not _ASTROPY_SETUP_:  # noqa
```

Comment:
I think all the `_ASTROPY_SETUP_` stuff can also be removed, no more useful since astropy-helpers removal, @astrofrog  ?

Reply:
Yes that's correct - the stuff inside the if statement should not be removed but doesn't need to be in an if statement anymore

Reply:
OK, I removed `_ASTROPY_SETUP_`. Please double check. Thanks!